app/utils.py

```diff
@@ -9,7 +13,17 @@ def accuracy_limit_to_percent(accuracy: int):
     return accuracy / 255 * 100
 
 
+def get_db():
```

Comment:
Could this move here: app/db/connect.py ?

Reply:
good one